libraries/chain/witness_schedule.cpp

```diff
@@ -8,18 +8,18 @@
 
 namespace sophiatx { namespace chain {
 
-void reset_virtual_schedule_time( database& db )
+void reset_virtual_schedule_time( const std::shared_ptr<database_interface>& db )
```

Comment:
scheduler is run only on full node, using database instead of datavase_interface is fine